azure-pipelines-micro.yml

```diff
@@ -0,0 +1,178 @@
+resources:
+  containers:
+    - container: ubuntu_x64_build_container
+      image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-20210311173856-047508b
+    - container: centos_x64_build_container
+      image: mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7-20220107135107-9b5bbc2
+
+# Trigger builds for PR's targeting main
+pr:
+  branches:
+    include:
+    - main
+  paths:
+    include:
+    - src/benchmarks/micro/*
+    - src/benchmarks/real-world/*
+    - src/harness/*
+    - src/tools/Reporting/*
+    - scripts/*
+    - eng/*
+    - azure-pipelines-micro.yml
+    exclude:
+    - scripts/benchmarks_monthly.py
+
```

Comment:
Does this need a schedule? I know it was on a schedule before and don't know if the files will update enough to trigger from here.